### PR TITLE
changes for succint

### PR DESCRIPTION
**Pull request by kroos47**
- [ ] Feature
- [ ] Bugfix
- [ ✔️ ] Refactor
- [ ] Format
- [ ] Documentation
- [ ] Testing
- [ ] Other:

## Description
 Header change for succint network which will be our default header. Swapping the appLookup and KateCommitment fields

### CODE REVIEW

File: avail-subxt/Cargo.toml [hunk 1]
```diff
@@ -35,6 +35,7 @@ serde-hex = "0.1"
 scale-info = { version = "2.5.0", features = ["bit-vec"] }
 
 
+
```
Comment (markopoloparadox) [inline avail-subxt/Cargo.toml:38]: This change is dangerous. I cannot allow it!

File: examples/ts/src/data_submit.ts [hunk 2]
```diff
@@ -53,7 +53,7 @@ async function main() {
                 const blockHash: H256 = result.status.asFinalized;
                 const exHash: H256 = result.txHash;
                 console.log(`Finalized block hash ${blockHash}`);
-                // get data from finalized block
+               // get data from finalized block
```
Comment (markopoloparadox) [inline examples/ts/src/data_submit.ts:56]: Weird formatting...

**Review by markopoloparadox — CHANGES_REQUESTED** [review]

File: avail-subxt/Cargo.toml (hunk 1 above)
Comment (kroos47) [reply avail-subxt/Cargo.toml:38]: oops! sorry boss for that change! reverting

**Review by markopoloparadox — APPROVED** [review]

**Review by Leouarz — APPROVED** [review]

**Review by 0xSasaPrsic — APPROVED** [review]
Looks good, we must test all dependencies after this merged.

**Pull request merged**